modules/thunder_updater/updates.yml

```diff
@@ -90,3 +90,7 @@ v1.1:
         '#title': 'New Thunder Admin UX'
         '#description': "<p>Thunder now comes with new improved administration user interface.</p>"
         '#description_failed': '<p><strong>There was a problem during the update.</strong> New Thunder Admin UX require plenty new configurations to be adjusted and imported in order to use it properly. We suggest you to take a look at execution of update hook for enabling new administration theme and manually adjust missing configurations.<p>'
+    v1_1__media_paragraph:
+        '#title': 'Updated media paragraph'
+        '#description': "<p>Last update of media paragraph to be compatible with thunder admin theme. The media paragraph is now considered deprecated and will not get any updates in the future.</p>"
```

Comment:
Should we emphasize (with <strong\> or so) that it's deprecated. Also text should be adjusted for "failed" case if we change logic in update hook.